libraries/Microsoft.Bot.Builder.Prompts/NumberPrompt.cs

```diff
@@ -54,30 +56,24 @@ protected NumberPrompt(IModel model, PromptValidator<NumberResult<T>> validator
                 {
                     if (float.TryParse(result.Resolution["value"].ToString(), out float value))
```

Comment:
Consider a variable name other than "value" here (out float value). Using keywords as variable names is confusing. It's also the same name as the method itself ("Value") save for the lower-case "v".